extensions/arc/src/ui/dashboards/postgres/postgresOverviewPage.ts

```diff
@@ -281,6 +342,54 @@ export class PostgresOverviewPage extends DashboardPage {
 		];
 	}
 
+	private getPodStatus(): PodStatusModel[] {
+		let podModels: PodStatusModel[] = [];
+		const podStatus = this._postgresModel.config?.status.podsStatus;
+
+		podStatus?.forEach(p => {
+			// If a condition of the pod has a status of False, pod is not Ready
+			const status = p.conditions.find(c => c.status === 'False') ? loc.notReady : loc.ready;
```

Comment:
So I guess I just wasn't sure if all conditions would have statuses that are true/false and whether there were other conditions that could be here that could be false but wouldn't necessarily mean the pod wasn't ready.

If that isn't the case this is probably fine - although I'm curious if there'd ever be a case where you have a condition whose status is False but there either isn't a 'Ready' condition with a 'True' status or no Ready condition at all. (Basically - are all conditions directly tied to the Ready condition such that if there's any that are false that means the pod for sure isn't ready? Is that something that's definitely not going to change?)

Reply:
It is possible the backend down the line could be improved to provide better calculations on the pods status. For now this is the best option to require all conditions to be met.